internal/services/network/network_manager_management_group_connection_resource.go

```diff
@@ -234,6 +234,34 @@ func (r ManagerManagementGroupConnectionResource) Delete() sdk.ResourceFunc {
 				return fmt.Errorf("deleting %s: %+v", id, err)
 			}
 
+			deadline, ok := ctx.Deadline()
+			if !ok {
+				return fmt.Errorf("context had no deadline")
+			}
+
+			// confirm the connection is fully deleted
```

Comment:
could we open a isue on the rest specs and link it here?

Reply:
link to https://github.com/Azure/azure-rest-api-specs/issues/23188